src/sqlitedb/database.hpp

```diff
@@ -228,23 +228,43 @@ namespace db
 
    public:
 
+    /// Prepares a query, caching it, and returns a wrapper that automatically resets the prepared
+    /// statement on destruction.
     StatementWrapper
     prepared_st(const std::string& query);
 
+    /// Prepares (with caching) and binds a query, returning the active statement handle.  Like
+    /// `prepared_st` the wrapper resets the prepared statement on destruction.
+    template <typename... T>
+    StatementWrapper
+    prepared_bind(const std::string& query, const T&... bind)
```

Comment:
are there any reasons that all these queries cant just be a string_view?

Reply:
Yes: sqlite3 requires a null-terminated C string, which string_view doesn't give you.